tests/tests.js

```diff
@@ -1,45 +1,177 @@
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+        coverageLevel: [{ role: 'ee', coverage: 125000 }]
+      }
+      const price = pricing.calculateVolLifePrice(
+        products.voluntaryLife,
+        selectedOptions
+      )
+      expect(price).to.equal(43.75)
+    })
+
+    it('return the price before employer contribution of the vol life product for employee and spouse', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee', 'sp'],
+        coverageLevel: [
+          { role: 'ee', coverage: 200000 },
+          { role: 'sp', coverage: 75000 }
+        ]
+      }
+      const price = pricing.calculateVolLifePrice(
+        products.voluntaryLife,
+        selectedOptions
+      )
+      expect(price).to.equal(79)
+    })
+
   })
+  describe('calculateVolLifePricePerRole', () => {
+    it('return the price for VolLife depending on the role', () => {
+      const coverageLevel = [{ role: 'ee', coverage: 125000 }]
+      const VolLifePrice = pricing.calculateVolLifePricePerRole(
+        'ee',
+        coverageLevel,
+        products.voluntaryLife.costs
+      )
 
-  it('returns the price for a disability product for an employee', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee']
-    }
-    const price = pricing.calculateProductPrice(products.longTermDisability, employee, selectedOptions)
+      expect(VolLifePrice).to.equal(43.75)
+    })
+  })
 
-    expect(price).to.equal(22.04)
+  describe(' calculateLTDPrice', () => {
+    it('return the price of ltd before employor contribution', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+      }
+      const LTDPrice = pricing.calculateLTDPrice(
+        products.longTermDisability,
+        employee,
+        selectedOptions
+      )
+      expect(LTDPrice).to.equal(32.04)
+    })
   })
+  describe('calculateProductPrice', () => {
+    let calculateLTDPriceSpy, formatPriceSpy, calculateVolLifePriceSpy, getEmployerContributionSpy, sandbox
+
+    beforeEach(() => {
+      sandbox = sinon.createSandbox()
+
+      formatPriceSpy = sandbox.spy(pricing, 'formatPrice')
+      calculateVolLifePriceSpy = sandbox.spy(pricing, 'calculateVolLifePrice')
+      calculateLTDPriceSpy = sandbox.spy(pricing, 'calculateLTDPrice')
+      getEmployerContributionSpy = sandbox.spy(pricing, 'getEmployerContribution')
+
+    })
+    afterEach(() => {
+      sandbox.restore()
+    })
+
+    it('returns the price for a voluntary life product for a single employee', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+        coverageLevel: [{ role: 'ee', coverage: 125000 }]
+      }
+      const price = pricing.calculateProductPrice(
+        products.voluntaryLife,
+        employee,
+        selectedOptions
+      )
+
+      expect(price).to.equal(39.37)
+      expect(getEmployerContributionSpy).to.have.callCount(1)
+
```

Comment:
You would also expect `calculateVolLifePriceSpy`, `calculateVolLifePricePerRoleSpy`, and `formatPriceSpy` all to be called once.